docs/source/02_get_started/04_hello_kedro.md

```diff
@@ -104,7 +105,14 @@ pipeline = Pipeline([return_greeting_node, join_statements_node])
 runner = SequentialRunner()
 
 # Run the pipeline
-runner.run(pipeline, data_catalog)
+print(runner.run(pipeline, data_catalog))
+```
+
+Then open a terminal and run the following command:
+
+```sh
+> python hello_kedro.py
+{'my_message': 'Hello Kedro!'}
```

Comment:
Nit: I think it would be more consistent with the rest of the documentation to not prefix this with `>`. However, other parts of the documentation don't show outputs, either. Maybe:

```suggestion
python hello_kedro.py
```

followed by something like: "You should see `{'my_message': 'Hello Kedro!'}`" printed to the console.